modules/installation/pages/server-setup.adoc

```diff
@@ -21,7 +21,7 @@ As part of providing product support, SUSE reserves the right to ask to remove a
 [IMPORTANT]
 ====
 Do not register a {productname} Server to itself.
-The {productname} Server must be managed individually.
+The {productname} Server must be managed individually or via another separate {productname} Server.
```

Comment:
```suggestion
The {productname} Server must be managed individually or by using another separate {productname} Server.
For more information about using multiple servers, see xref:large-deployments:multi-server.adoc[].
```

Reply:
Also: this statement exists in multiple locations, we'll need to update them globally.

Reply:
> Also: this statement exists in multiple locations

Fixed